### PR TITLE
Documentation for Cake for Rider release 0.3.0

### PR DESCRIPTION
**Pull request by nils-a**
Documentation of the upcoming release of Cake for Rider 0.3.0

### CODE REVIEW

**Review by gep13 — APPROVED** [review]
LGTM!

**Comment by nils-a** [comment]
[Cake for Rider 0.3.0](https://github.com/cake-build/cake-rider/releases/tag/0.3.0) has been submitted and is now in moderation. I will merge this PR once the package habe been approved by JetBrains.

**Pull request merged**
